tensorflow/contrib/cmake/README.md

```diff
@@ -181,7 +181,16 @@ Step-by-step Windows build
    More? -Dtensorflow_ENABLE_GPU=ON ^
    More? -DCUDNN_HOME="D:\...\cudnn"
    ```
-    
+   To enable SIMD instructions with MSVC, as AVX and SSE, define them as follows:
+   ```
+   More? -Dtensorflow_WIN_CPU_SIMD_OPTIONS=/arch:AVX
+   ```     
+   or if more than one as in:
+   ```
+   More? -Dtensorflow_WIN_CPU_SIMD_OPTIONS=/arhc:[AVX2|SSE2]
```

Comment:
Oops, it was very late, will fix the typo :)

I had the same reading of this syntax. When looking at [1] it seems that is how you define the set of instructions but apparently you are right. I tested on x64 which only supports AVX and AVX2 and using both it yelled at me. Would you please confirm it does not work?

I will change it anyway leaving only the first option `/arch:ARG`, what do you think?

[1] https://msdn.microsoft.com/en-us/library/7t5yh4fd.aspx